test/Microsoft.Dnx.Testing.CommonUtils/DnxFunctionalTestBase.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class DnxSdkFunctionalTestBase
+    {
+        public const string SdkVersionForTestingEnvName = "DNX_SDK_VERSION_FOR_TESTING";
+
+        static DnxSdkFunctionalTestBase()
+        {
```

Comment:
Oh, don't do this. Static constructors are usually a bad idea, you have no idea when this will be invoked